Changelog.md

```diff
@@ -4,6 +4,8 @@ Language Features:
 
 
 Compiler Features:
+* Commandline Interface: Add `--no-cbor-metadata` that skips CBOR metadata from getting appended at the end of the deployed bytecode.
+* Standard JSON: Add a boolean field `settings.metadata.appendCBOR` that skips CBOR metadata from getting appended at the end of the deployed bytecode.
```

Comment:
This is not inconsistent with https://github.com/ethereum/solidity/pull/13265#discussion_r940130991, isn't it?
```suggestion
* Commandline Interface: Add `--no-cbor-metadata` that skips CBOR metadata from getting appended at the end of the bytecode.
* Standard JSON: Add a boolean field `settings.metadata.appendCBOR` that skips CBOR metadata from getting appended at the end of the bytecode.
```